content/docs/_index.md

```diff
@@ -13,11 +13,22 @@ material for all [supported languages](languages/).
 
 - **Eager to see gRPC in action?**
 
-  [Select a language](languages/) and then choose its **Quick start**.
+  Select a [language][] or [platform][], then choose its **Quick start**.
 
 - Interested in **gRPC feature details?**
 
   Try one of the following:
 
-  - [Select a language](languages/) and then choose **Tutorial** or **API reference**
+  - Select a [language][] or [platform][], then choose **Tutorial** or **API reference**
```

Comment:
This should just need to be `[lanugage] or [platform]`.

Reply:
If you write `[text][]` then some editors (like VS Code) give edit-time validation that your markdown link is defined (by underlining it, in the case of VS Code). Very useful feature. As you point out, it doesn't affect the link generation.

Reply:
Neat! I'm going to have to start doing this

Reply:
`[text][]` is part of original markdown. I didn't realize `[text]` worked. That sounds like a markdown extension, which is why some editors might not notice it.